t/140-ssl-c-api.t

```diff
@@ -811,3 +817,700 @@ lua ssl server name: "test.com"
 --- no_error_log
 [error]
 [alert]
+
+
+
+=== TEST 6: simple cert + private key with password
```

Comment:
FWIW, the pattern i've been using is testing the C FFI function in `lua-nginx-module` and test the Lua function in `lua-resty-core`. Otheriwse you create a co-dependency on both repos that makes the tests fail for both lua-nginx-module and lua-resty-core PRs until you merge both. (Assuming this is what you meant in the first paragraph).